### ISSUE TITLE
LTN_Tracker::on_gui_click depot_data nill

### ISSUE THREAD

**Issue by bkendall** [eduran84/LTN_Tracker#13]
**Describe the bug**
Twice now I've had my headless server crash and emit the following error:

```
90087.371 Error MainLoop.cpp:1035: Exception at tick 23739552: Error while running event LTN_Tracker::on_gui_click (ID 1)
__LTN_Tracker__/ui/depot_tab.lua:249: attempt to index local 'depot_data' (a nil value)
90087.372 Error ServerMultiplayerManager.cpp:96: MultiplayerManager failed: "Error while running event LTN_Tracker::on_gui_click (ID 1)
__LTN_Tracker__/ui/depot_tab.lua:249: attempt to index local 'depot_data' (a nil value)"
90087.372 Info ServerMultiplayerManager.cpp:699: mapTick(23739552) changing state from(InGame) to(Failed)
90087.385 Info GlobalContext.cpp:696: Waiting for child processes to exit:
```

AFAIK, a user had clicked on the button to show the UI and the server crashes. I do not have explicit replication steps yet, but I wanted to get this on your radar. I'm going to keep my eye out for it again and try to replicate, but don't hold your breath waiting :)

**Desktop (please complete the following information):**
Version `0.8.3` running on a headless `0.16` server.

Let me know if there's anything else I can provide for you.

**Comment by eduran84** [comment]
Any chance someone was renaming depot stops around that time? If so it should be fixed with 0.8.5 (which is not released, yet).

**Comment by bkendall** [comment]
There's a good chance that was the case. I'll keep an eye on it after
updating to `0.8.5` (soon)
